apps/storybook/.storybook/preview.js

```diff
@@ -34,6 +37,38 @@ export const globalTypes = {
       title: 'Color mode',
     },
   },
+  locale: {
+    name: 'Locale',
+    description: 'Internationalization locale',
+    defaultValue: 'en',
+    toolbar: {
+      icon: 'globe',
```

Comment:
This is super unnecessary, but wdyt about changing the icon we use for "Color mode" to `'paintbrush'`, as I agree that `"globe"` makes sense for "Locale".

Completely unnecessary, but having different icons for the two different controls makes sense to me 😄

[List of supported icons](https://main--64b56e737c0aeefed9d5e675.chromatic.com/?path=/docs/introduction--docs)

Reply:
Yeah maybe in future 👍 But if we change it, I'd rather use the same one as product primer for consistency:

<img width="79" height="37" alt="Screenshot 2025-07-10 at 15 56 01" src="https://github.com/user-attachments/assets/cd7ddfbb-e4f0-46da-8694-2808755ae9dc" />